umbraco-cloud/set-up/project-settings/README.md

```diff
@@ -113,13 +114,9 @@ You can change your Umbraco Cloud project to run in a dedicated setup with addit
 
 On your Umbraco Cloud project, it is possible to see the usage of Custom Domains, Media Storage, Content Nodes, and Bandwidth for the project. You can also check if it is using above or below the allowed amount for the plan that your project is on.
 
-## [Payment](../manage-subscriptions/new-shop.md)
-
-From here, you can see the payment history for your project, change the credit card, as well as download the latest invoices for your project.
```

Comment:
@sajumb any reason why you are deleting this?

Reply:
The page has been removed. Now the user will see only be able to see this information through the organization page.

Reply:
Is that also the case for users in the old shop, or is it just for users in the new one?

Reply:
Good point. I might have been too quick as I am not aware of this 😕  I will consult KEC and come back to you on this one.

Reply:
Great! We also did updated the article for payments in the new shop to show that it is paid through the organisation, so I think we should keep them for now, and then we can have a look at moving the article for the new shop to the organisation section instead :)

Reply:
KEC states that we should remove the reference to the payment stuff on the project level. That should not be a thing anymore, as we do collective billing and not billing per project. Also for projects in the old shop.

Reply:
Okay, great :) 

Thanks for clarifying, then it's fine that we remove it :) 

Make sure to also remove it from the SUMMARY.md file :)